cloud/aws/services/ec2/instances.go

```diff
@@ -33,6 +36,8 @@ const (
 
 	// InstanceStatePending indicates the instance is pending
 	InstanceStatePending = ec2.InstanceStateNamePending
+
+	defaultAMIID = "ami-057c58c0eca3e6fe3"
```

Comment:
💯

Reply:
Another note here is that AMIs are region-specific, so this might end up failing in a different region.

Reply:
yeah, took that into account with the new logic. It's **very** bad right now though because the AMIs don't have tags to filter on, but it's fine for now.